src/sentry/api/endpoints/rule_snooze.py

```diff
@@ -104,20 +106,25 @@ def post(self, request: Request, project: Project, rule: Rule | AlertRule) -> Re
 
         if not can_edit_alert_rule(project.organization, request):
             raise PermissionDenied(
-                detail="Requesting user cannot mute this rule.", code=status.HTTP_403_FORBIDDEN
+                detail="Requesting user cannot mute this rule.", code=str(status.HTTP_403_FORBIDDEN)
```

Comment:
the `str(403)` seems suspicious -- what was this fixing? do we even need `code=` (surely a PermissionDenied is always going to be a 403 right?)

Reply:
yeah, good pt. the 403 comes included in the `PermissionDenied` exception class already